pkg/cosign/cosign.go

```diff
@@ -101,10 +101,10 @@ func VerifySignature(opts Options) (digest string, err error) {
 	if err != nil {
 		msg := err.Error()
 		log.Info("image verification failed", "error", msg)
```

Comment:
@Namanl2001 - looks like the Cosign messages have improved and we can directly use them. I would suggest removing the logic to customize the message, and simply return the error from Cosign to match what the user would see when they run `cosign validate`.

Reply:
verification for unsigned image returns the below error, and it's a bit misleading 
```
cosign verify --key cosign.pub ghcr.io/kyverno/test-verify-image:unsigned
Error: no matching signatures:

main.go:46: error during command execution: no matching signatures:
```

expected: `signature not found` as it's an unsigned image. Thanks

Reply:
Is this with the current Cosign CLI version? Ivan reported a different message on the download command:

> Error: no signatures associated with 10.1.0.12:5000/busybox:latest

Reply:
on my cli also download command is giving the same err msg:

```
cosign download signature ghcr.io/kyverno/test-verify-image:unsigned
Error: no signatures associated with ghcr.io/kyverno/test-verify-image:unsigned
main.go:46: error during command execution: no signatures associated with ghcr.io/kyverno/test-verify-image:unsigned
```

let me check with the latest cosign-cli

Reply:
checked! 
getting the same error msg for the `unsigned` image as mentioned in my second last comment i.e. (`Error: no matching signatures`)